pkg/knuu/knuu.go

```diff
@@ -176,9 +193,8 @@ func setDefaults(ctx context.Context, k *Knuu) error {
 		k.Logger = log.DefaultLogger()
 	}
 
-	if k.TestScope == "" {
-		t := time.Now()
-		k.TestScope = fmt.Sprintf("%s-%03d", t.Format("20060102-150405"), t.Nanosecond()/1e6)
+	if k.TestScope == "" && k.K8sClient != nil {
+		k.TestScope = k.K8sClient.Namespace()
```

Comment:
You're missing the default base case of setting TestScope when both are empty. 
```suggestion
	if k.TestScope == "" && k.K8sClient != nil {
		k.TestScope = k.K8sClient.Namespace()
	} else if k.TestScope == "" {
		t := time.Now()
		k.TestScope = fmt.Sprintf("%s-%03d", t.Format("20060102-150405"), t.Nanosecond()/1e6)
```

Reply:
well it was like this before, but in one of the comments you told me to move this default test scope to the e2e pkg: 
https://github.com/celestiaorg/knuu/pull/434/files/41fff2e769f1e40f1a336d265a1579265647fc47#diff-04a1accd1c12dafffe8d3ecc43a623cd0bf22d03d3561f7e04e296e64962048eR8-R11
that's the reason for receiving it from the user. I am more fan of making it easier for the users, so will put it back.

Reply:
Sorry, the name of this variable and the function is confusing. 
When it was refactored it looked like a testing variable, which is why I made the original comment. 
Knuu is a Test framework, which makes sense why this is called TestScope. I see that now. 

Maybe in a follow up we could rename this to just be Scope to avoid future confusion

Reply:
yeah that's a good idea to rename it.